manifests/manifests-servicemesh/servicemesh.package.yaml

```diff
@@ -0,0 +1,8 @@
+packageName: servicemeshoperator
+channels:
+- name: "1.0"
+  currentCSV: servicemeshoperator.v1.1.0
+channels:
```

Comment:
same as for the maistra.package.yaml: two `channels` and the `1.0` channels points to 1.1.0

Reply:
Same answers as above. ;)